### PR TITLE
my 1st PR

### PR DESCRIPTION
**Pull request by DaveKhach**
Added some scenarios on createDepartment, added request() method, added response specs.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,6 +45,11 @@
       <artifactId>groovy</artifactId>
       <version>3.0.2</version>
     </dependency>
+    <dependency>
```
Comment (SandroDarbinyan) [inline pom.xml:48]: Actually we do not use this dependency, why we need to add that?

File: src/Files/config.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+baseUri="http://192.168.88.14:8800/api/"
```
Comment (SandroDarbinyan) [inline src/Files/config.txt:1]: There is an existing solution for this on main branch. Do not need to have this.

File: src/test/java/com/qtlab/automation/test/sms/Department/GetDepartmentTest.java [hunk 3]
```diff
@@ -23,14 +27,21 @@ public void createRequestSpecification() {
                 build();
     }
 
+    @BeforeEach
```
Comment (SandroDarbinyan) [inline src/test/java/com/qtlab/automation/test/sms/Department/GetDepartmentTest.java:30]: There is a beforeEach hook, why do you need to create second beforeEach block

File: pom.xml (hunk 1 above)
Comment (DaveKhach) [reply pom.xml:48]: Added to be used, but didn't use it yet. Will add next when will be used.

File: src/test/java/com/qtlab/automation/test/sms/Department/GetDepartmentTest.java (hunk 3 above)
Comment (DaveKhach) [reply src/test/java/com/qtlab/automation/test/sms/Department/GetDepartmentTest.java:30]: That is for responseSpec.. method. Will it accept in beforeEach hook if the second annotation take back?

**Pull request closed without merging**